dist/vokal-ng-api.js

```diff
@@ -187,7 +187,7 @@ angular.module( "vokal.API", [ "vokal.Humps" ] )
         };
         apiConstruct.prototype.setKey = function ( key )
         {
-            this.globalHeaders.AUTHORIZATION = String( key );
+            this.globalHeaders.AUTHORIZATION = key ? String( key ) : "";
```

Comment:
Well yeah checking for undefined/null is probably too wide a net if the only goal is to allow 0, maybe just something like `key || ( !isNaN( key ) && isFinite( key ) )`.